src/vst_plugin/RnNoiseVstPlugin.cpp

```diff
@@ -41,5 +41,55 @@ void RnNoiseVstPlugin::getProgramName(char *name) {
 }
 
 extern AudioEffect *createEffectInstance(audioMasterCallback audioMaster) {
-    return new RnNoiseVstPlugin(audioMaster, 0, 0);
+    return new RnNoiseVstPlugin(audioMaster, 0, RnNoiseVstPlugin::numParameters);
+}
+
+void RnNoiseVstPlugin::getParameterLabel(VstInt32 index, char* label) {
+    const auto paramIdx = static_cast<Parameters>(index);
+    switch (paramIdx) {
+        case Parameters::vadThreshold:
+            strcpy(label, paramVadThresholdLabel);
+            break;
+    }
+}
+
+void RnNoiseVstPlugin::getParameterName(VstInt32 index, char* label) {
+    const auto paramIdx = static_cast<Parameters>(index);
+    switch (paramIdx) {
+        case Parameters::vadThreshold:
+            strcpy(label, paramVadThresholdName);
+            break;
+    }
+}
+
+void RnNoiseVstPlugin::getParameterDisplay(VstInt32 index, char* label) {
+    const char* formatStr = "%.3f";
+    int bufferLength = 6; // The length of the value expressed with formatStr
+    char buffer [bufferLength];
+    
+    const auto paramIdx = static_cast<Parameters>(index);
+    switch (paramIdx) {
+        case Parameters::vadThreshold:
+            snprintf(buffer, bufferLength, formatStr, paramVadThreshold);
```

Comment:
I think you could directly format into `label` without additional buffer, also you could directly specify format string. And as a bufferLength you should set `VstStringConstants::kVstMaxParamStrLen`.